Example/Tests/UnsplashKitSpec.swift

```diff
@@ -0,0 +1,23 @@
+import Foundation
+import Quick
+import Nimble
+
+@testable import UnsplashKit
+
+extension SourceRequestFactory: Equatable {}
+public func ==(lhs: SourceRequestFactory, rhs: SourceRequestFactory) -> Bool {
+    return lhs === rhs
+}
+
+
+class UnsplashKitSpec: QuickSpec {
+    override func spec() {
+        
+        let requestFactory = SourceRequestFactory()
+                
+        describe("-init") {
```

Comment:
We should never test if we set the attributes in a constructor. You can remove this test.